pkg/trace/agent/tags.go

```diff
@@ -280,14 +284,22 @@ func NormalizeTag(tag string) string {
 			continue
 		}
 
-		c = unicode.ToLower(c)
+		if utf8.ValidRune(c) && unicode.IsUpper(c) {
+			// lowercase this character
+			if low := unicode.ToLower(c); utf8.RuneLen(c) == utf8.RuneLen(low) {
+				// but only if the width of the lowercased character is the same;
+				// there are some rare edge-cases where this is not the case, such
+				// as \u017F (ſ)
+				utf8.EncodeRune(norm[i:], low)
```

Comment:
Should we also do `c = low` so any further operations with `c ` actually look at the new character we put in its place?

Reply:
👍